### ISSUE TITLE
🧐 Functionalities of Ant Design Charts

### ISSUE THREAD

**Issue by alexroginski1** [ant-design/ant-design-charts#1453]
Hello,
I have a few questions about functionalities of Ant Design Charts:
1. Is it possible to show annotations in the tooltip?
2. Is it possible to have multiple charts on one plot. For example, can you have an area chart and a line chart in the same plot?
3. How can you handle long bar labels on bar charts?
4. Is there a way to fix the error shown in the console when you create onClick events for points on a Line chart, for example?
Thanks!

**Comment by lxfu1** [comment]
Yes, it is recommended to see the example first,  communication if there is a problem.

**Comment by alexroginski1** [comment]
The examples in the Gallery do not show those functionalities. I could not find anything about these problems in the documentation or online.
Are the four functionalities listed available currently in Ant Design Charts?

**Comment by alexroginski1** [comment]
I would still appreciate guidance on the questions posted above.
Also, how do plot layers work? Can I use them to stack plots on top of each other? I am most interested in having a line chart and area chart on the same plot. This was the only reference I could find: https://charts.ant.design/en/docs/map-api/layers/area-layer
